docs/config.json

```
@@ -603,6 +603,10 @@
             {
               "title": "Host User Creation",
               "slug": "/server-access/guides/host-user-creation/"
+            },
+            {
+              "title": "EC2 Instance Discovery",
```

@lxea This should also be marked as a preview feature.
```suggestion
              "title": "EC2 Instance Discovery (Preview)",
```